src/Exporter/AssetCallableExporter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Assets\Exporter;
+
+use Yiisoft\Assets\AssetExporterInterface;
+
+/**
+ * Exports asset bundles with the values of all properties using a custom PHP callable.
+ */
+final class AssetCallableExporter implements AssetExporterInterface
```

Comment:
Isn't it better to have your own implementation in a class with a strict interface rather than having it in a callable?